docs/ldml/tr35-personNames.md

```diff
@@ -612,7 +612,16 @@ For example:
 3. Pattern C is discarded, because it has the least number of populated name fields.
 4. Out of the remaining patterns A and B, pattern B wins, because it has only 3 unpopulated fields compared to pattern A.
 
-#### 6.5.2 <a name="process-the-namepattern" href="#process-the-namepattern">Process a namePattern</a>
+#### 6.5.2 <a name="missing-surname" href="#missing-surname">Handle missing surname</a>
+
+All PersonName objects will have a given name (for mononyms the given name is used). However, there may not be a surname. In that case, the following process is followed so that formatted patterns produce reasonable results.
+
+1. If there is no surname from a PersonName P1 _and_ there is no non-initialized given name in the pattern, then
```

Comment:
"non-initialized" reads weird to me, because I want to use the standard computer-science meaning of "initialize".  I'd suggest using more words here: "If there is no surname from a PersonName P1 and the pattern either doesn't include the given name or only shows an initial for the given name, then..."